the-experts-BE/database/db_build.sql

```diff
@@ -10,6 +10,28 @@ BEGIN;
         job TEXT NOT NULL,
         rating double precision	 DEFAULT 0
     );
+
+    CREATE TABLE
+    IF NOT EXISTS Users
+    (
+        id SERIAL PRIMARY KEY,
+        first_name VARCHAR
+    (50) NOT NULL,
+            last_name VARCHAR
+    (50) NOT NULL,
+        email TEXT NOT NULL,
+        user_password TEXT NOT NULL
+    );
+
+INSERT INTO Users
+    (first_name,last_name,email,user_password)
+VALUES
+    ('ebraheem', 'abbas', 'ebraheemabbas51@gmail.com', 'password');
```

Comment:
The password you insert is going to be hashed right? You should insert the hashed version of 'password' into table.